docs/stylesheets/extra.css

```
@@ -12,12 +11,12 @@
   display: flex;
   align-items: center;
 
-  padding-right: 0;     
-  margin-right: 0;      
+  padding-right: 0;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)